capella2polarion/connectors/polarion_worker.py

```diff
@@ -98,11 +90,11 @@ def load_polarion_work_item_map(self):
         """Return a map from Capella UUIDs to Polarion work items."""
         work_items = self.client.get_all_work_items(
             "HAS_VALUE:uuid_capella",
-            {"workitems": "id,uuid_capella,checksum,status"},
+            {"workitems": "id,uuid_capella,checksum,status,type"},
```

Comment:
Why are we adding `type` here now?

Reply:
We need the type when we insert work items in documents. Otherwise we won't be able to support layouts based on the type of a work item